tests/Kernel/MediaRenderTest.php

```diff
@@ -250,4 +254,31 @@ public function testDocumentMedia(): void {
     }
   }
 
+  /**
+   * Tests that Iframe media is rendered inside media container ECL component.
+   */
+  public function testIframeMedia(): void {
+    // Create a Iframe media without defined aspect ratio.
+    $media = $this->mediaStorage->create([
+      'bundle' => 'iframe',
+      'name' => 'test iframe',
+      'oe_media_iframe' => '<iframe src="http://example.com/iframe_media"></iframe>',
+    ]);
+    $media->save();
+
+    $build = $this->mediaViewBuilder->view($media, 'oe_theme_main_content');
+    $html = $this->renderRoot($build);
+    $crawler = new Crawler($html);
+    $iframe = $crawler->filter('.ecl-media-container .ecl-media-container__media--ratio-16-9 iframe');
```

Comment:
As per specifications, if no ratio is specified in the iframe media, no ratio should be used in the media container.

Reply:
ECL component components/ec-component-media-container/ecl-media-container.html.twig sets default ratio 16:9 if ratio isn't provided.
There isn't class that defines media container without ratio.
If we pass in ratio variable value `no-ratio` then class is `ecl-media-container__media--ratio-no-ratio` and media container looks broken:
<img width="1366" alt="Screenshot 2020-12-10 at 11 12 43" src="https://user-images.githubusercontent.com/1183951/101739663-e983ad80-3ad8-11eb-9af3-7841a48a51eb.png">

Therefore if we want not to set any ratio ECL's media container should be adjusted.

Reply:
Well if the media container is meant only to handle responsiveness, then it's as easy as not using it when a custom iframe size is needed. Let's change the code this way.

Reply:
@brummbar sorry, but what does "Let's change the code this way." mean?